config/routes.rb

```diff
@@ -3,6 +3,9 @@
 
   # Defines the root path route ("/")
   resources :users, only: [:show, :index] do 
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:create]
```

Comment:
- [ ] As per the comments controller actions `new` and `create`, kindly ensure that is a pair of corresponding routes in the `config/routes.rb` file for creating comments, which should yield:
```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
 GET   /posts/:post_id/comments/new(.:format)     comments#new
```